tools/restatectl/src/commands/node/remove_nodes.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::commands::node::disable_node_checker::DisableNodeChecker;
+use crate::connection::ConnectionInfo;
+use anyhow::Context;
+use clap::Parser;
+use cling::{Collect, Run};
+use itertools::Itertools;
+use restate_cli_util::c_println;
+use restate_core::metadata_store::{MetadataStoreClient, Precondition};
+use restate_metadata_server::create_client;
+use restate_types::PlainNodeId;
+use restate_types::config::{MetadataClientKind, MetadataClientOptions};
+use restate_types::metadata_store::keys::NODES_CONFIG_KEY;
+use restate_types::nodes_config::{NodesConfiguration, Role};
+
+#[derive(Run, Parser, Collect, Clone, Debug)]
+#[clap(alias = "rm")]
+#[cling(run = "remove_nodes")]
+pub struct RemoveNodesOpts {
+    /// The node/s to remove from the cluster. Specify multiple nodes as a comma-separated list or
+    /// specify the option multiple times.
+    #[arg(long, required = true, visible_alias = "node", value_delimiter = ',')]
+    nodes: Vec<PlainNodeId>,
+}
+
+pub async fn remove_nodes(
+    connection: &ConnectionInfo,
+    opts: &RemoveNodesOpts,
+) -> anyhow::Result<()> {
+    let nodes_configuration = connection.get_nodes_configuration().await?;
+    let metadata_client = create_metadata_client(&nodes_configuration).await?;
+    let logs = connection.get_logs().await?;
```

Comment:
Does get_logs() check is this is not an empty logs with INVALID version, or should we validate here that this invariant holds?

Reply:
`get_logs()` will only return a `Logs` if its version is > `Version::INVALID`.